docs/mutect/mutect.tex

```diff
@@ -236,6 +236,26 @@ \section{Germline Filter}\label{germline-filter}
 
 To filter, we set a threshold on this posterior probability.
 
+\section{Contamination Filter}\label{contamination-filter}
+Suppose our tumor bam has contamination fraction $\alpha$ and that at some site we have $a$ alt reads out of $d$ total reads.  Suppose further that the alt allele has population allele frequency $f$.  We will compute a simple estimate of the posterior probability that these alt reads came from a contaminating sample and not from a true somatic variant.  Letting $\pi$ be the prior probability of somatic variation as above a very crude model for the alt count distribution of somatic variation is a uniform distribution.  That is, we assume that any value of $a$ from $0$ to $d$ is equally likely.  Then the likelihood of the data given a true somatic variant is
+\begin{equation}
+P(a | {\rm somatic}) = \frac{\pi}{d + 1}.
```

Comment:
Isn't the likelihood P(a | somatic) equal to 1/(d+1), and the joint probability P(a, somatic) = pi/(d+1)?

Reply:
True.